src/components/clusterConfiguration/ClusterConfiguration.tsx

```diff
@@ -269,7 +189,7 @@ const ClusterConfiguration: React.FC<ClusterConfigurationProps> = ({ cluster })
                       </TextContent>
                       <InputField label="Cluster Name" name="name" isRequired />
                       <InputField
-                        label="Base DNS domain"
+                        label="Base DNS Domain"
```

Comment:
I think we are mixing these capitalized and non capitalized labels throughout the screens. We should pass through all strings and unify them